chatbot-ui/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './App.css';
+import ChatInterface from './components/ChatInterface';
+
+function App() {
```

Comment:
Great start on the app. Let's use cursor / some designs to make it look a bit nicer. You can use vercel AI editor/website builder to give you some ideas.